cost-analyzer/values.yaml

```diff
@@ -932,6 +932,43 @@ kubecostDeployment:
     configVolumeSize: 1Gi
     initImage: {}
 
+kubecostAggregator:
+  cloudCost:
+    enabled: false
+  replicas: 0
+  resources: {}
+  env:
+    "LOG_LEVEL": "info"
+  persistentConfigsStorage:
+    # default storage class
+    storageClass: ""
+    storageRequest: 1Gi
+  aggregatorStorage:
+    # default storage class
+    storageClass: ""
+    storageRequest: 20Gi
+  aggregatorDbStorage:
+    # default storage class
+    storageClass: ""
+    storageRequest: 128G
```

Comment:
Standardize on units of gibibytes here (Gi)

Reply:
good catch, fixed